app/views/pages/_terms_text.html.erb

```diff
@@ -1,4 +1,4 @@
-    <h3 id="terms">
+    <h3 id="terms-text">
```

Comment:
doesn't this change anything?

Reply:
There were two elements with this Id on the page. :sweat_smile:

Reply:
Are the IDs used for anything? It's not form data as they're `<h3 />` tags, so my guess is they aren't being used, unless they're being used with hashbangs, e.g. https://dev.to/somePage#terms

If they serve no purpose, I would just remove them.